include/mxnet/gpu_swap_history.h

```diff
@@ -0,0 +1,58 @@
+#ifndef GPU_SWAP_HISTORY_H_
+#define GPU_SWAP_HISTORY_H_
+
+#include <chrono>
+#include <vector>
+#include <thread>
+#include <mutex>
+#include "./swap.h"
+
+#if MXNET_USE_CUDA
+#include <cuda_runtime.h>
+#endif
+
+#define NUMBER_OF_GPU 8
```

Comment:
Based on google coding style, I think it is better to use const variable instead of define